Automattic/wp-calypso
# Login: Support Jetpack color scheme for 2FA screens

#### Changes proposed in this Pull Request

* Login: Support Jetpack color scheme for 2FA screens

#### Preview

2FA push notification screen (before):
![](https://cldup.com/p56gbXBkaq.png)

2FA push notification screen (after):
![](https://cldup.com/ueSC34k37P.png)

2FA authenticator screen (before):
![](https://cldup.com/cpUB1z9hAG.png)

2FA authenticator screen (after):
![](https://cldup.com/LNaiLsfrIC.png)

Re design review: I realize these testing instructions are complex, so glancing at the screenshots should be enough for now. There are a couple of minor things like border and text colors, but we can fix these in subsequent PRs. The purpose of this PR is to reuse the same Jetpack color scheme we already have in other screens, for the 2FA screens.

#### Testing instructions
* Checkout this branch on your local Calypso.
* Start Calypso (`npm start`)
* Make sure you're logged out of WP.com.
* Go to http://calypso.localhost:3000/log-in/jetpack/
* Log in with an account that has 2FA enabled.
* Verify that the 2FA screens have the Jetpack color scheme.
* Verify you're able to complete the log in process.
* Go to http://calypso.localhost:3000/log-in/
* Verify that the 2FA screens **don't** have the Jetpack color scheme.
* Verify you're able to complete the log in process.
* Verify you're able to log in as a user that has 2FA disabled.
* Try this with the new Jetpack connect-in-place flow:
  * Start a new JN site with the latest bleeding edge Jetpack: https://jurassic.ninja/create?jetpack-beta
  * Enable the iframe constant from the constants settings page in your JN site.
  * SSH into the site using the command in the JN site admin notification.
  * Navigate to the WP installation dir by calling `cd DIR` where `DIR` is the path as specified in the JN site admin notification.
  * `vim wp-config.php`
  * `:i` to go to INSERT mode.
  * Add `define( 'CALYPSO_ENV', 'development );` somewhere where the other constants are.
  * Press `esc` key to leave INSERT mode.
  * `:w` to save.
  * `:q` to quit.
  * Make sure you're logged out of WP.com.
  * Go to `/wp-admin/admin.php?page=jetpack#/dashboard` in the JN site.
  * Connect the site for a WP.com user that has 2FA enabled.
  * Verify you can see the Jetpack color scheme for the 2FA screens, so they're consistent with the login screen.

Fixes https://github.com/Automattic/jetpack/issues/13419
